frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -75,6 +75,17 @@ class ChartSettings extends Component {
         return assocIn(this.props.series, [0, "card", "visualization_settings"], this.state.settings);
     }
 
+    getChartTypeName(){
+      if (this.props.series[0].card.display === "table") {
+        return "table";
+      } else if (this.props.series[0].card.display === "scalar"){
+        return "number";
+      } else {
+        return "chart";
+      } 
+    }
```

Comment:
NBD but for next time, we're using 4 spaces for indentation, not 2

I think we need to add an entry to our `.editorconfig` file for `.jsx` files. Right now we just have one for `.js` (@kdoh?)